bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/BreadcrumbImpl.java

```diff
@@ -48,6 +48,7 @@
     protected static final boolean PROP_SHOW_HIDDEN_DEFAULT = false;
     protected static final boolean PROP_HIDE_CURRENT_DEFAULT = false;
     protected static final int PROP_START_LEVEL_DEFAULT = 2;
+    private static final boolean PROP_DISABLE_SHADOWING = false;
```

Comment:
Please add `_DEFAULT` to constant name so it's easier to understand what it's used for!